classes/mender-sdimg.bbclass

```diff
@@ -86,7 +86,7 @@ IMAGE_CMD_sdimg() {
     mkfs.vfat "${WORKDIR}/boot.vfat"
 
     # Copy boot files to boot partition
-    for file in ${IMAGE_BOOT_FILES}
+    for file in "${IMAGE_BOOT_FILES}"
```

Comment:
This won't work, because it may contain multiple entries, and we expect it to be split on whitespace. Why doesn't it work for you?